### ISSUE TITLE
Error loading class com.elmakers.mine.bukkit.math.dSin: com.elmakers.mine.bukkit.math.dSin

### ISSUE THREAD

**Issue by mibby** [elBukkit/MagicPlugin#235]
Magic dev 2146
Paper dev 1104 (Spigot 1.11.2)

Trying to use the boomerang spell results in this error occurring in console. Nothing happens but a bunch of particles sitting still until they fade away.

```
[08:22:57] [Server thread/WARN]: Error loading class com.elmakers.mine.bukkit.math.dSin: com.elmakers.mine.bukkit.math.dSin
```

**Comment by NathanWolf** [comment]
Hm, seems broken. I'll take a look.

**Comment by NathanWolf** [comment]
Fixed in dev build#2153

http://jenkins.elmakers.com/job/MagicPlugin/2153/com.elmakers.mine.bukkit.plugins$Magic/

**Comment by NathanWolf** [comment]
Note that boomerang is not really a finished spell, which is why I don't use it in my own configs anywhere.
